charts/osm/grafana/dashboards/osm-mesh-envoy-details.json

```diff
@@ -73,7 +73,7 @@
       "pluginVersion": "7.0.1",
       "targets": [
         {
-          "expr": "osm_k8s_monitored_namespace_count{source_pod_name=~\"$osm_controller_instance\"}",
+          "expr": "sum(osm_k8s_api_event_count{type=\"namespace-added\", source_pod_name=~\"$osm_controller_instance\"}) by (source_pod_name) OR on() vector(0) - sum(osm_k8s_api_event_count{type=\"namespace-deleted\", source_pod_name=~\"$osm_controller_instance\"}) by (source_pod_name) OR on() vector(0)",
```

Comment:
Has this been tested with Grafana?

Reply:
@shashankram the namespace counter is being incremented correctly, but while testing I realized that the existing counters for namespaces, services, and pods are not being decremented for some reason when the corresponding resources are deleted. I will open an issue and fix this in a separate PR.

Reply:
Let's fix this before merging this change, because it would otherwise break functionality.